ci/deps/travis-37-aarch64.yaml

```diff
@@ -0,0 +1,22 @@
+name: pandas-dev
+channels:
+  - defaults
+  - conda-forge
+  - c3i_test
+dependencies:
+  - python=3.7.*
+
+  # tools
+  - cython>=0.29.13
```

Comment:
the error message for this build says cython is missing

Reply:
Hi  @jbrockmendel ,
I've tried explicitly installing cython through "setup_env.sh" script. But even then it's throwing the error .
Also, i have a doubt: If i mention a dependency in .yaml file in this directory, how does the build system installs that dependency? or does it even install that dependency ? In my findings, .yaml file only checks if the mentioned dependencies are present or not .

Can you please shed light on this ?

thanks

Reply:
my understanding is that when you set ENV_FILE in travis.yml, that tells conda what to install.

Reply:
for the compilation failure, your guess is as good as mine